src/Files.App/Actions/Content/Tags/OpenAllTaggedAction.cs

```diff
@@ -0,0 +1,34 @@
+﻿using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Files.App.Helpers;
+using Files.Shared.Extensions;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Files.App.Actions.Content.Tags
+{
+	internal class OpenAllTaggedAction : IAction
+	{
+		private readonly ITagsContext tagsContext = Ioc.Default.GetRequiredService<ITagsContext>();
+		
+		private readonly IContentPageContext pageContext = Ioc.Default.GetRequiredService<IContentPageContext>();
+
+		public string Label => "OpenAllItems".GetLocalizedResource();
+
+		public RichGlyph Glyph { get; } = new("\uE8E5");
```

Comment:
Which icon is this?

Reply:
It's a temp icon since `OpacityIcon` is not working
![1](https://user-images.githubusercontent.com/102259289/224557217-45e6b852-b6d9-4448-880a-350de573c904.png)

Reply:
Can you try `&#xE71D;`?